aep/general/0148/aep.md.j2

```diff
@@ -1,5 +1,155 @@
 # Standard fields
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/28) for more
-information.
+Certain concepts are common throughout any corpus of APIs. In these situations,
+it is useful to have a standard field name and behavior that is used
+consistently to communicate that concept.
+
+## Guidance
+
+Standard fields **should** be used to describe their corresponding concept, and
+**should not** be used for any other purpose.
+
+### Resource paths and IDs
+
+#### path
+
+Every resource **must** have a `string path` field, used for the resource path
+(AEP-122), which **should** be the first field in the resource.
+
+#### parent
+
+The `string parent` field refers to the resource name of the parent of a
+collection, and **should** be used in most `List` (AEP-132) and `Create`
+(AEP-133) requests.
+
+### Names
+
+#### display_name
+
+The `string display_name` field **must** be a mutable, user-settable field
```

Comment:
not that we'll probably run into this case, but there may be existing display names that serve a different purpose - like a name that is user-facing vs internal.

do we need this to be a must?

Reply:
Isn't this always a user-facing rather than internal name?

Are you suggesting that sometimes display_name might need to be immutable or not user-settable, and so this must should be a should?

Reply:
I was thinking that the field might already be used for a different purpose. Although I guess that's somewhat unlikely.

But immutability is a good one - if an API already came in with display_name that is immutable, would we call it non-compliant and not provide tools for it? probably not.

Reply:
Discussed on today's call. Will change to `should` and then this can be merged.

Reply:
Done.